index.ts

```diff
@@ -1,4 +1,4 @@
-export * from './lib/config';
+export type { Config } from './lib/config';
+export type { Message } from './lib/message';
```

Comment:
Previously we were exporting `*`, but now we're only exporting `Config` and `Message`. Just want to make sure it's intentional that we're no longer exporting [`MessageAttributes`](https://github.com/mycase/chulie/blob/52bb08162ddaf7a494fac1b44cd528d2844c330e/lib/message.d.ts#L4), [`MessageBody`](https://github.com/mycase/chulie/blob/52bb08162ddaf7a494fac1b44cd528d2844c330e/lib/message.d.ts#L8), [`MessageBodyFormat`](https://github.com/mycase/chulie/blob/52bb08162ddaf7a494fac1b44cd528d2844c330e/lib/message.d.ts#L10), [`QueueDriveMode`](https://github.com/mycase/chulie/blob/57a2d5d8437dd95dc4841868cb49a8eb1a037d57/lib/config.d.ts#L4), [`AwsConfig`](https://github.com/mycase/chulie/blob/57a2d5d8437dd95dc4841868cb49a8eb1a037d57/lib/config.d.ts#L6), [`QueueConfig`](https://github.com/mycase/chulie/blob/57a2d5d8437dd95dc4841868cb49a8eb1a037d57/lib/config.d.ts#L12), and [`MessageConfig`](https://github.com/mycase/chulie/blob/57a2d5d8437dd95dc4841868cb49a8eb1a037d57/lib/config.d.ts#L20).

Reply:
yes, we only need to expose `Config` and `Message` to end user.